lib/src/screens/auth/register/register.dart

```diff
@@ -0,0 +1,110 @@
+// ignore_for_file: unnecessary_const
+
+import 'package:flutter/cupertino.dart';
+
+class Register extends StatelessWidget {
```

Comment:
Название классов для страниц следует называть с суффиксом `Screen`.
```suggestion
class RegisterScreen extends StatelessWidget {
```

Такое название даёт сразу понимание о том, за что отвечает данный класс. Так же, название файла тоже должно соответствовать названию класса. Поэтому файл тоже должен называться: `register_screen.dart`